{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"name":"group17_10.ipynb","provenance":[],"collapsed_sections":[],"authorship_tag":"ABX9TyONutqIqXX0ww7EmNmaJ/cx"},"kernelspec":{"name":"python3","display_name":"Python 3"},"language_info":{"name":"python"}},"cells":[{"cell_type":"code","metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"eKgO7ntnbVWZ","executionInfo":{"status":"ok","timestamp":1636012023932,"user_tz":-330,"elapsed":4001,"user":{"displayName":"Jessie James","photoUrl":"https://lh3.googleusercontent.com/a/default-user=s64","userId":"17242250360897825904"}},"outputId":"23162ec6-0c84-41b1-dce0-6ceb0b72e65f"},"source":["\"\"\" 10. Find error(s) in the following code(if any) and rewrite code. \"\"\"\n","try:\n","  x = int(input(\"Enter value of x:\"))\n","  flag=0\n","  for y in range(0,10):     \n","     if(x==y):\n","          flag=1\n","          print(x,\"is present in the range of values from 0 to 9\")\n","          break\n","  if(flag==0):\n","         print(x,\"is not present in the range of values from 0 to 9\")\n","except ValueError:\n","  print(\"That was not a number!\")"],"execution_count":6,"outputs":[{"output_type":"stream","name":"stdout","text":["Enter value of x:9\n","9 is present in the range of values from 0 to 9\n"]}]}]}